roles/foreman/tasks/main.yaml

```diff
@@ -0,0 +1,115 @@
+---
+- name: Pull the Foreman container image
```

Comment:
We haven't done this yet with the other services -- but I tend to agree I like the explicit pull instead of the quadlet doing it.